pkg/cli/manifest/testclientfactory.go

```diff
@@ -0,0 +1,160 @@
+/*
+You may obtain a copy of the License at
+
```

Comment:
Same :) please check the license headers on the new files. https://github.com/radius-project/radius/pull/8120/files#diff-f0158862504865933773e70cd591585504ad7dcd16bda2dde458f967c3901fbeL2